examples/mobile/src/main/java/org/apache/apex/examples/mobile/Application.java

```diff
@@ -158,7 +158,7 @@ public void populateDAG(DAG dag, Configuration conf)
     LOG.info("Finished generating seed data.");
 
     String gatewayAddress = dag.getValue(DAG.GATEWAY_CONNECT_ADDRESS);
-    URI uri = URI.create("ws://" + gatewayAddress + "/pubsub");
+    URI uri = URI.create((dag.getValue(DAG.GATEWAY_USE_SSL) ? "wss://" : "ws://") + gatewayAddress + "/pubsub");
```

Comment:
Should this be in util class?

Reply:
Yes, however, since it is also being used in core, I have added it there and it will take a release to be available here.